src/trading-api/next/draft/bullish-trading-api.yml

```diff
@@ -0,0 +1,6049 @@
+        entryNotional:
+          description: Notional of the position in currency of the quote asset when entering the position
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        mtmPnl:
+          description: Mark to Market PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        reportedMtmPnl:
+          description: Reported Mark to Market PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        reportedFundingPnl:
+          description: Report Funding PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        realizedPnl:
+          description: Realized PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        settlementAssetSymbol:
+          description: Settlement Asset Symbol
+          type: string
+          example: "USD"
+        createdAtDatetime:
+          description: Denotes the time the position was created by the exchange, ISO 8601 with millisecond as string
+          allOf:
+            - $ref: '#/components/schemas/DateTime'
+        createdAtTimestamp:
+          description: Denotes the time the position was created by the exchange, number of milliseconds since EPOCH
+          allOf:
+            - $ref: '#/components/schemas/TimeStampAsString'
+        updatedAtDatetime:
+          description: Denotes the time the position was updated by the exchange, ISO 8601 with millisecond as string
+          allOf:
+            - $ref: '#/components/schemas/DateTime'
+        updatedAtTimestamp:
+          description: Denotes the time the position was updated by the exchange, number of milliseconds since EPOCH
+          allOf:
+            - $ref: '#/components/schemas/TimeStampAsString'
+
+    PerpetualSettlementResponse:
+      description: Perpetual Settlement of one market for the trading account
+      type: string
+      properties:
+        tradingAccountId:
+          allOf:
+            - $ref: '#/components/schemas/TradingAccountId'
+        symbol:
+          example: "BTC-USD-PERP"
+          allOf:
+            - $ref: '#/components/schemas/MarketSymbol'
+        side:
+          allOf:
+            - $ref: '#/components/schemas/OrderSide'
+        settlementQuantity:
+          allOf:
+            - $ref: '#/components/schemas/AssetValue'
+        deltaTradingQuantity:
+          allOf:
+            - $ref: '#/components/schemas/AssetValue'
+        deltaAmmQuantity:
+          allOf:
+            - $ref: '#/components/schemas/AssetValue'
+        mtmPnl:
+          description: Mark to Market PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        fundingPnl:
+          description: Funding PnL
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        markPriceAtSettlement:
+          description: Mark Price
+          allOf:
+            - $ref: '#/components/schemas/UsdValue'
+        indexPriceAtSettlement:
```

Comment:
I think we should standardize this naming convention. We have `atSettlement` and `settledAt`. I prefer if we do `settlementIndexPrice`

Reply:
So the fields should all be `settlement *`